revscores/datasources/is_content_namespace.py

```diff
@@ -0,0 +1,11 @@
+from ..datasources import namespaces, revision_metadata
+from .feature import Feature
+
+def process(revision_metadata, namespaces):
+    
+    content_namespace_ids = set(ns.id for ns in namespaces.values() if ns.content)
+    
+    return revision_metadata.page_namespace in content_namespace_ids
+
+is_content_namespace = Feature("is_content_namespace", process, returns=bool,
```

Comment:
I believe this should be at revscores/features/is_content_namespace.py.
Also, isn't the new feature supposed to be a replacement for revscores/features/is_mainspace.py, to fix issue #8?